lambda/update/main.go

```diff
@@ -17,10 +16,17 @@ type Logger interface {
 	Print(...interface{})
 }
 
+type Store interface {
+	Put(ctx context.Context, data any) error
+	Get(ctx context.Context, uid string) (shared.Lpa, error)
+}
+
 type Lambda struct {
-	store    shared.Client
-	verifier shared.JWTVerifier
-	logger   Logger
+	store    Store
+	verifier interface {
+		VerifyHeader(events.APIGatewayProxyRequest) bool
+	}
```

Comment:
In the other lambdas this is still `verifier shared.JWTVerifier`, did this one need to be changed?

Reply:
Yeah, because I wanted to inject a mock version in the tests. I'll name the type though to make it look nicer